src/main/java/net/neoforged/neoforge/common/world/chunk/TicketController.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.common.world.chunk;
+
+import java.util.UUID;
+import javax.annotation.ParametersAreNonnullByDefault;
+import net.minecraft.core.BlockPos;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.level.ForcedChunksSavedData;
+
+/**
+ * A class used to manage chunk loading tickets associated with a specific ID.
+ * <p>
+ * Controllers must be registered via {@link RegisterTicketControllersEvent}. A controller that isn't registered will have all of its tickets discarded, and the chunks will no longer be forcefully loaded.
```

Comment:
This is unclear. Do you mean when the world is reloaded? Or do all attempts to force load chunks just fail? If the latter, that should be asserted instead of just ignored imo.